tests/app/confirmation_page/test_confirmation_page.py

```diff
@@ -1,19 +1,21 @@
 from app.questionnaire.navigator import Navigator
+from app.data_model.answer_store import Answer, AnswerStore
 from app.schema_loader.schema_loader import load_schema_file
 import unittest
 
 
 class TestConfirmationPage(unittest.TestCase):
 
-    def setUp(self):
-        survey = load_schema_file("0_rogue_one.json")
+    def test_get_next_location_confirmation(self):
+        current_block_id = "5e633f04-073a-44c0-a9da-a7cc2116b937"
```

Comment:
Sure, I can do that. This isn't a new test though - simply an updated one to use new navigator init approach.